test/parallel/test-http-client-spurious-aborted.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+const common = require('../common');
+const http = require('http');
+const assert = require('assert');
+const fs = require('fs');
+const Countdown = require('../common/countdown');
+
+function cleanup(fname) {
+  try {
+    if (fs.statSync(fname)) fs.unlinkSync(fname);
+  } catch (err) {}
+}
+
+const N = 2;
+const fname = '/dev/null';
+let abortRequest = true;
+
+const server = http.Server(common.mustCall((req, res) => {
+  const headers = { 'Content-Type': 'text/plain' };
+  headers['Content-Length'] = 50;
+  const socket = res.socket;
+  res.writeHead(200, headers);
+  setTimeout(() => res.write('aaaaaaaaaa'), 100);
```

Comment:
Shouldn't all of these timers be `setTimeout(...., common.platformTimeout(n))` instead?